source/index.html.md.erb

```diff
@@ -34,12 +46,14 @@ are still working for us as an organization.
 
 ### Operating services
 
-- [Managing DNS](standards/dns-hosting.html)
-- [Monitoring your service](standards/monitoring.html)
-- [Operating systems](standards/operating-systems.html)
-- [Responding to problems](standards/alerting.html)
-- [Sending emails](standards/sending-email.html)
-- [Storing and querying logs](standards/logging.html)
+- [managing DNS](standards/dns-hosting.html)
+- [monitoring your service](standards/monitoring.html)
+- [operating systems](standards/operating-systems.html)
+- [responding to problems](standards/alerting.html)
+- [sending emails](standards/sending-email.html)
+- [storing and querying logs](standards/logging.html)
+- [securing a service](standards/securing-services.html)
+- [supporting a service](standards/supporting-services.html)
```

Comment:
can we keep this list in alphabetical order?

also, if we're adding new standards does this change need to be approved by tech forum?